src/mame/drivers/gigatron.cpp

```diff
@@ -81,7 +81,7 @@ void gigatron_state::gigatron(machine_config &config)
 
 	/* video hardware */
 	screen_device &screen(SCREEN(config, "screen", SCREEN_TYPE_RASTER));
-	screen.set_refresh_hz(59.98);
+	screen.set_refresh_hz(59.94);
```

Comment:
I don’t consider this “fixed” if you haven’t actually derived screen timings from the dot clock etc.  Also, unrelated changes shouldn't be mixed up like this.

Reply:
https://gigatron.io/?page_id=482

"Video: 59.98 Hz VGA compatible (standard DE-15 connector)"

Reply:
So is it actually 59.98 or 59.94?  I haven’t looked at the schematics myself, but I’d bet it isn’t actually using the same crystal and dividers as a real VGA card, so it would have different timing parameters.  You need to actually identify the timing circuitry and derive the video parameters from that.

Consider this source file: https://github.com/mamedev/mame/blob/mame0217/src/mame/drivers/osborne1.cpp

There are three machines, each with different screen timing parameters.  Two of them (`osborne1` and `osborne1sp`) derive the dot clock and raster timings from the master clock that’s also divided down for CPU and RAM timing.  The third one (`osborne1nv`) uses a programmable CRT controller with its own crystal.  Note that in all three cases, the screen parameters can be set in terms of the dot clock, using the `set_raw` configuration function.

You have schematics for this machine.  Use them.